pkg/docker/util.go

```diff
@@ -286,6 +286,28 @@ func GetBuilderImage(config *api.Config) (*PullResult, error) {
 	return r, nil
 }
 
+// GetRebuildImage obtains the metadata information for the image
+// specified in a s2i rebuild operation.  Assumptions are made that
+// the build is available locally since it should have been previously built.
+func GetRebuildImage(config *api.Config) (*PullResult, error) {
+	d, err := New(config.DockerConfig, config.PullAuthentication)
+	if err != nil {
+		return nil, err
+	}
+
+	r, err := PullImage(config.Tag, d, api.PullNever, false)
```

Comment:
thinking about it some more, I'll reuse the builder policy ... the tester for example specified the -p option ... seems natural